packages/surveys/src/utils.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { Model } from "survey-jquery";
+import {
+  consentSurveyFunction,
+  exitSurveyFunction,
+  textMarkdownSurveyFunction,
+} from "./utils";
+
+jest.mock("@lookit/data", () => ({
+  ...jest.requireActual("@lookit/data"),
+  updateResponse: jest.fn().mockReturnValue("Response"),
+}));
+
+test("", () => {
```

Comment:
Should this test have a name?

Reply:
Yup, all tests should have names.  Thanks.